pkg/parser/parser.go

```diff
@@ -0,0 +1,135 @@
+package parser
```

Comment:
add the header boilerplate

Reply:
I checked pombump and infra-images. We didn't add the boilerplate :).

Reply:
Maybe we can move on without.

Reply:
ok no worries i can do in a follow up